src/omnisharp/server.ts

```diff
@@ -258,16 +304,16 @@ export class OmniSharpServer {
                 launchPath = await this._omnisharpManager.GetOmnisharpPath(this._options.path, this._options.useMono, serverUrl, latestVersionFileServerPath, installPath, extensionPath);
             }
             catch (error) {
-                this.eventStream.post(new OmnisharpFailure(`Error occured in loading omnisharp from omnisharp.path\nCould not start the server due to ${error.toString()}`, error));
+                this.eventStream.post(new ObservableEvents.OmnisharpFailure(`Error occured in loading omnisharp from omnisharp.path\nCould not start the server due to ${error.toString()}`, error));
                 return;
             }
         }
 
-        this.eventStream.post(new OmnisharpInitialisation(new Date(), solutionPath));
+        this.eventStream.post(new ObservableEvents.OmnisharpInitialisation(new Date(), solutionPath));
```

Comment:
Why do we need to add the `ObservableEvents` specifier everywhere? It makes the code much more busy..

Reply:
The import from logging events was becoming too long so I changed that to `import * as ObservableEvents from loggingEvents` . What do you think is better ?